src/main/java/com/google/api/generator/gapic/model/RegionTag.java

```diff
@@ -138,7 +138,8 @@ public String generate() {
     if (!apiVersion().isEmpty()) {
       rt = rt + apiVersion() + "_";
     }
-    rt = rt + "generated_" + serviceName() + "_" + rpcName();
+    rt = rt.toLowerCase() + "generated_" + serviceName() + "_" + rpcName();
+    // TODO: @alicejli overloadDisambiguation will need to be updated to match other languages
```

Comment:
Is this a must have for Phase 1?

Reply:
No - to be fair I think the overloadDisambiguation hasn't been formalized across languages in the design; I just happened to notice Java's doesn't look the same as C# (which I'm taking as canonical). I'm going to keep as a TODO to check on later, but right now I think it's okay that Java is different.